app/chaturbate/worker.go

```diff
@@ -93,75 +94,79 @@ func statRoom(chQuit chan struct{}, room, server, proxy string, info *tID, u url
 		}
 	}
 
+	defer func() {
```

Comment:
if we must do some stuff in all cases when exit from func - best place to add defer statement in head of func